addons/mrp_account/views/product_views.xml

```diff
@@ -5,7 +5,6 @@
             <field name="model">product.template</field>
             <field name="priority">3</field>
             <field name="inherit_id" ref="product.product_template_only_form_view" />
-            <field name="groups_id" eval="[(4, ref('mrp.group_mrp_manager'))]"/>
```

Comment:
Surely there isn't any problem keeping this level of permissions in this views. The issue came indeed from raising the level on the `stock_account` ones.

By the way, a less breaking fix to the original issue would be to add a new module with a new permission to cost update actions. Something like this: https://github.com/OCA/product-attribute/tree/13.0/stock_account_product_cost_security

Reply:
And for those who update already, the view will remain with the former permissions, so they should be manually removed (or you could add a migration script that ensures the right permissions).

Reply:
Yes, I know adding permission for this view, it does not create an issue, but it is better to delete the whole commit to avoid another bug. Users can add it manually via settings

Adding a new module, fields, view is not allowed in a stable odoo version.

The commit was merged this week, so it is not deployed on the client DB yet. But if a customer has already done the update, he can create a help ticket, and we will fix it manually in his DB

Reply:
Ok, thaks Djamel :)